content/appleboy/drone-gitlab-ci/index.md

```diff
@@ -1,71 +1,51 @@
 ---
-version: '0.8'
-date: 2017-07-14T00:00:00+00:00
+date: 2019-19-19T00:00:00+00:00
 title: Gitlab-CI
 author: appleboy
 tags: [ infrastructure, trigger, gitlab, gitlab-ci ]
-logo: gitlab.svg
 repo: appleboy/drone-gitlab-ci
+logo: gitlab-ci.svg
```

Comment:
There is no gitlab-ci.svg, but there is a gitlab.svg

Reply:
@tboerger Done